updater/scripts/update-changelog.ps1

```diff
@@ -60,6 +79,26 @@ for ($i = 0; $i -lt $lines.Count; $i++)
         continue
     }
 
+    # Skip the prettier comment that may be found before the Unreleased version.
```

Comment:
Too bad this needs to be duplicated... I wonder why I didn't just start this loop where the previous one ended... Anyway, that's not related to this PR so feel free to leave as is and resolve this comment

Reply:
I also found it odd too but I'd rather to just support this feature for the moment than refactoring the script. Maybe afterwards when I have more free time I could refactor it but for the moment the code is working